### PR TITLE
Mainnet Deploy Governance Contracts

### PR DESCRIPTION
**Pull request by LondonCalamari**
Script for deploying the governance contracts to mainnet

### CODE REVIEW

File: scripts/deployGovernanceMainnetContracts.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  console.log("deploying with:", deployer.address);
+
+  const proxyAdmin = await deployAndValidate("ProxyAdmin", deployer, "ProxyAdmin", []);
+
+  const governanceStrategy = await deployAndValidate("LyraGovernanceStrategy", deployer, "LyraGovernanceStrategy", [
+    LYRA,
+    STK_LYRA,
+  ]);
+
+  const GUARDIAN = "0x246d38588b16Dd877c558b245e6D5a711C649fCF";
```
Comment (0xdomrom) [inline scripts/deployGovernanceMainnetContracts.ts:22]: maybe just move this to the top of the file

File: scripts/deployGovernanceMainnetContracts.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    // grace period - time after `delay` while a proposal can be executed
+    5 * DAY_SEC,
+    // min delay
+    DAY_SEC,
+    // max delay
+    10 * DAY_SEC,
+    // propositionThreshold (in percentage of 10000, so 100 = 1% of total tokens to create proposal)
+    50,
+    // vote duration (blocks, so ~3day)
+    17280, // number of blocks vote lasts after the voting delay
```
Comment (0xdomrom) [inline scripts/deployGovernanceMainnetContracts.ts:44]: You use 12sec blocks for the governance contract above, but 15 sec blocks here. This should be "21386" if using 12.12 sec blocks (or 21600 if using 12sec)

File: scripts/deployGovernanceMainnetContracts.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    7 * DAY_SEC,
+    // grace period - time after `delay` while a proposal can be executed
+    5 * DAY_SEC,
+    // min delay
+    7 * DAY_SEC,
+    // max delay
+    10 * DAY_SEC,
+    // propositionThreshold (in percentage of 10000, so 100 = 1% of total tokens to create proposal)
+    200,
+    // vote duration (blocks, so ~10 days)
```
Comment (0xdomrom) [inline scripts/deployGovernanceMainnetContracts.ts:65]: with 12sec block times this would be ~8 days

File: test/voting.t.ts [hunk 4]
```diff
@@ -146,6 +146,29 @@ describe("GovernorBravo voting with stkLyra", function () {
...
+
+    await skipBlocks(6);
+    await aaveGovernance.connect(alice).submitVote(1, true);
+    await skipBlocks(10);
+    await aaveGovernance.connect(admin).queue(1);
+    await fastForward(8 * DAY_SEC);
+    await aaveGovernance.connect(admin).execute(1);
+
+    expect(await executor.getDelay()).eq(8 * DAY_SEC);
   });
```
Comment (0xdomrom) [inline test/voting.t.ts:172]: could you add to this test a vote using the TransferEth contract and delegateCall flow?

**Review by 0xdomrom — CHANGES_REQUESTED** [review]

File: scripts/deployGovernanceMainnetContracts.ts (hunk 2 above)
Comment (LondonCalamari) [reply scripts/deployGovernanceMainnetContracts.ts:44]: Updated to 21600 (12sec blocks)

File: scripts/deployGovernanceMainnetContracts.ts (hunk 3 above)
Comment (LondonCalamari) [reply scripts/deployGovernanceMainnetContracts.ts:65]: Updated to 72000 (12sec blocks)

**Review by 0xdomrom — APPROVED** [review]
lgtm

**Pull request merged**